lib/beaker-puppet/helpers/puppet_helpers.rb

```diff
@@ -673,12 +673,12 @@ def stub_forge_on(machine, forge_host = nil)
         #                             global options hash
         def with_forge_stubbed_on( host, forge_host = nil, &block )
           #use global options hash
-          primary_forge_name = 'forge.puppetlabs.com'
+          primary_forge_name = 'forgeapi.puppet.com'
           forge_host ||= options[:forge_host]
           forge_ip = resolve_hostname_on(host, forge_host)
           raise "Failed to resolve forge host '#{forge_host}'" unless forge_ip
           @forge_ip ||= forge_ip
-          with_host_stubbed_on( host, {primary_forge_name => @forge_ip}, {primary_forge_name => ['forge.puppet.com','forgeapi.puppetlabs.com','forgeapi.puppet.com']}, &block )
+          with_host_stubbed_on( host, {primary_forge_name => @forge_ip}, {primary_forge_name => ['forge.puppet.com','forgeapi.puppetlabs.com','forge.puppetlabs.com']}, &block )
```

Comment:
Is this line correct?

Reply:
@binford2k Is there something in particular that looks wrong to you? My understanding was that this should include all the aliases of the `primary_forge_name` - if I've changed the `primary_forge_name` to 'forgeapi.puppet.com', so it doesn't need to be an alias anymore, but the old value for `primary_forge_name` does. Is that not right? I'm not super familiar with beaker stuff.

Also I don't really know the difference between this method and `stub_forge_on`, but I made the same changes to both.